app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:id])
+    @posts = Post.where(user_id: @user.id)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = Post.where(user_id: @user.id)
+    @post = Post.find(params[:post_id])
+  end
```

Comment:
- [ ] Kindly fix the show action for post(s) post. I have posts I created on the Rails console, when I try to fetch a specific post using `http://localhost:3000/users/1/posts/1` I got an error.

![image](https://user-images.githubusercontent.com/53124548/216293419-809349c5-b2b3-4a27-88b8-de9fdf1fb8a7.png)